datalad/distribution/install.py

```diff
@@ -321,21 +322,48 @@ def __call__(dataset=None, path=None, source=None, recursive=False,
 
         # resolve the target location against the provided dataset
         if path is not None:
-            path = resolve_path(path, ds)
+            # make sure it is not a URL, `resolve_path` cannot handle that
+            if is_url(path):
+                try:
+                    path = get_local_path_from_url(path)
+                    path = resolve_path(path, ds)
```

Comment:
this line is apparently not test covered -- could there be a test for this case as well? ;)